### ISSUE TITLE
Spotify for Artists API

### ISSUE THREAD

**Issue by jonathonpitman** [spotify/web-api#1035]
Is there any future plans to open the API up to allow for 3rd party apps to access things like play counts for the artist that is currently logged in?

**Comment by villeodell** [comment]
Great idea.. would be nice to extend a similar set of options to record labels also.

**Comment by bradleywyros** [comment]
Any updates on this? It would be amazing for artists to have access to their Spotify For Artists insights via API much like users are able to. For example, any artist profile that a user has access to, if they log in with their Spotify credentials, an API would be able to access the Artists they are associated with's insights.

**Comment by anandijain** [comment]
I would love to be able to access my stats from an API!

**Comment by vinyldarkscratch** [comment]
+1 for this.  On my web app, I’d like to offer linkage to a Spotify Artist profile through OAuth or some other method that verifies the account actually belongs to that user.

**Comment by nathanchere** [comment]
There is already a private analytics API (the 'Standard Bulk API') for getting data about steams and listeners, but for some reason the entire existence of the API is intentionally obscured.

Aside from the complete lack of acknowledgement of requests like this one speaking volumes in itself, they are only moving further away from making this information easily and openly available to artists with things like the GDPR and recent Californian Consumer Protection Act changes.

Basically: don't count on this happening soon/ever.
